### PR TITLE
Tabs style

### PR DESCRIPTION
**Pull request by jonkiky**
## Description

Update tabs and cart's style

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,6 +1,6 @@
 REACT_APP_BACKEND_GETUSERINFO_API=https://k9dc.essential-dev.com/fence/login/
 REACT_APP_LOGIN_URL=https://nci-crdc-staging.datacommons.io/user/oauth2/authorize?client_id=82pslYFJqA7auRvKYfTOK67jzQAMb8f6C33tlmZz&response_type=code&redirect_uri=https%3A%2F%2Fk9dc.essential-dev.com%2F&scope=openid%20user
 REACT_APP_USER_LOGOUT_URL=https://k9dc.essential-dev.com/fence/logout
-REACT_APP_BACKEND_API=http://localhost:8080/v1/graphql/
+REACT_APP_BACKEND_API=https://api-dev.bento-tools.org/v1/graphql/
```
Comment (skiran86) [inline .env:4]: Please revert this change

**Review by skiran86 — CHANGES_REQUESTED** [review]

**Comment by jonkiky** [comment]
function and style have been updated.  Ready for review.

File: src/pages/dashboardTab/components/tabConfigs/fileConfig.js [hunk 2]
```diff
@@ -102,10 +102,10 @@ export function fileColumns(classes) {
       label: 'Program Code',
       options: {
         filter: false,
-        customBodyRender: (value) => (
+        customBodyRender: (value, tableMeta) => (
           <div className="mui_td" style={tableStyle(0.8)}>
             {' '}
-            <Link to={`/case/${value}`} className={classes.link}>{value}</Link>
+            <Link to={`/program/${tableMeta.rowData[2]}`} className={classes.link}>{value}</Link>
```
Comment (AjayDoddapaneni) [inline src/pages/dashboardTab/components/tabConfigs/fileConfig.js:108]: Make sure this value is not hardcoded during custodian as the custodian can change  order

File: src/pages/dashboardTab/components/tabController.js [hunk 3]
```diff
@@ -40,6 +90,49 @@ const tabController = (classes) => {
 && state.dashboardTab.datatable
     ? state.dashboardTab.datatable : {}));
 
+  const tooltipContent = {
+    0: 'Click button to add selected files associated with selected cases(s)',
```
Comment (AjayDoddapaneni) [inline src/pages/dashboardTab/components/tabController.js:94]: During custodian try cases, samples, and files instead of 0,1,2

**Review by AjayDoddapaneni — APPROVED** [review]
Can you address the above comments during custodian.

**Comment by AjayDoddapaneni** [comment]
Can you fix the codacy issue before merging

File: src/pages/dashboardTab/components/tabController.js (hunk 3 above)
Comment (jonkiky) [reply src/pages/dashboardTab/components/tabController.js:94]: 0,1,2 are more generic as tab may not use for case, sample and files in the future.

**Review by AjayDoddapaneni — CHANGES_REQUESTED** [review]
@jonkiky Unable to run this branch on local, eslint is choking on errors

**Review by AjayDoddapaneni — APPROVED** [review]
